src/RuleSet.php

```diff
@@ -530,6 +565,13 @@ private function getSetDefinition($name)
             throw new \InvalidArgumentException(sprintf('Set "%s" does not exist.', $name));
         }
 
+        if (\PHP_VERSION_ID < 70100) {
```

Comment:
I don't think presets should be dependant on the runtime PHP version. It need not be the same as the minimum source target version.

Reply:
We agree. This needed #4946 which is merged now so I removed the condition.